### PR TITLE
Add tiny-js-world task

### PR DESCRIPTION
**Pull request by misshatsker**
### Tiny JS World

[Demo](https://misshatsker.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/misshatsker/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/misshatsker/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+const inhabitants = [
+   woman,
+   man,
+   dog,
+   cat
+];
+
+function getInhabitantInfo(inhabitant) {
+   return `${inhabitant.species}; ${inhabitant.name}; ${inhabitant.gender}; ${inhabitant.legs}; ${inhabitant.hands}; ${inhabitant.saying}`;
```
Comment (artem-trubin) [inline submissions/misshatsker/tiny-js-world/index.js:51]: This is a pattern here, the code is repeated. Imagine a situation where you would have more properties and you had a task to change the separator from `;` to, let's say, `,`. Find a way to specify keys by hand and then cycle through them getting information from objects. Make sure to **NOT** use `Object.values` and similar methods as they aren't reliable in terms of key order and presence.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Hey, nice job. Though you went a little ahead and remade your demo for the OOP version already, I'm sure the pre-oop demo worked just fine. There is still one improvement there is to make here, check the comment below.

File: submissions/misshatsker/tiny-js-world/index.js (hunk 1 above)
Comment (misshatsker) [reply submissions/misshatsker/tiny-js-world/index.js]: Thanks for the review! I've improved it, Could you please check it again?

**Review by artem-trubin — APPROVED** [review]
Great job!

**Pull request merged**